helper-cli/src/main/kotlin/commands/GeneratePackageConfigurationsCommand.kt

```diff
@@ -21,19 +21,30 @@ package org.ossreviewtoolkit.helper.commands
 
 import com.github.ajalt.clikt.core.CliktCommand
 import com.github.ajalt.clikt.parameters.options.convert
+import com.github.ajalt.clikt.parameters.options.flag
```

Comment:
I cannot make any sense of the commit message body text. Are you somehow continuing the sentence from the title? That's confusing, IMO, and the body text should make sense on its own.

Reply:
I'm continuing the sentence from the title, yes. Thought it was common practice here in ORT. Let me change it...

Reply:
Actually, I believe it's uncommon practice (both in ORT and in general; you also don't continue from a book's title in the first chapter). And it was really not obvious to me as you were starting the commit message body with a capital letter.